src/jaegertracing/TracerFactoryTest.cpp

```diff
@@ -65,19 +67,70 @@ TEST(TracerFactory, testValidConfig)
         "refreshInterval": 60
     }
   })";
-    TracerFactory tracerFactory;
+    const auto tracerFactory = new (std::nothrow) TracerFactory(true);
```

Comment:
There is a memory leak here. There is no need to call new. TracerFactory tracerFactory(new) is sufficient.